src/java.base/unix/native/libjli/java_md_solinux.h

```diff
@@ -60,4 +66,23 @@ static const char *user_dir     = "/java";
 #include <pthread.h>
 #endif
 
+#ifdef AIX
+#define ZLIBNX_PATH "/usr/opt/zlibNX/lib"
+
+#ifndef POWER_9
+#define POWER_9     0x20000     /* 9 class CPU */
+#define POWER_9_ANDUP   (POWER_9)
+#define __power_9_andup() (_system_configuration.implementation & POWER_9_ANDUP)
+#endif
```

Comment:
why not add POWER10 as well here?
we don't want to rework this, since P10 support was added on OpenJ9/OMR side.

Reply:
I'm assuming that P10 or newer would have `POWER_9` defined. Given that this is sufficient and doesn't require a rework for any future POWER CPU.

Reply:
Apparently not, see https://github.com/eclipse/omr/blob/master/port/unix/omrsysinfo.c#L233-L241.

Reply:
Fixed in 0c38988

Reply:
wrong fix.  you may break the build actually.  POWER_9 definition doesn't mean POWER_10_ANDUP is defined too.